src/gimelstudio/core/project_file.py

```diff
@@ -144,8 +148,11 @@ def WriteFile(self, file_path, contents):
 
     def OpenFile(self, file_path):
         with open(file_path, "r") as file:
-            file_contents = json.load(file)
-            self.contents = file_contents
+            try:
+                file_contents = json.load(file)
+                self.contents = file_contents
+            except Exception as e:
+                wx.MessageBox(message=_("Unable to create graph from file, the file maybe corrupt\n(Details: check 2 failed to complete:)"))
```

Comment:
@Correct-Syntax Not sure entirely what this message should read.

Again, ideally the details text would contain more information.

Reply:
> Sorry for being quite pedantic here. Just a couple of little things to update, and then it should be all good. slightly_smiling_face

No worries dude, we should make sure it's done right.

Reply:
Should be all set except for that one you tagged Correct Syntax on

Reply:
Still working on getting the project file I/O working again. I think we'll merge this PR first, once it's ready.